### PR TITLE
T5-284 Update Ebook form

### PR DESCRIPTION
**Pull request by naeluh**

### CODE REVIEW

File: static/functions/submission-created.js [hunk 1]
```diff
@@ -443,19 +445,16 @@ exports.handler = async (event, _context, callback) => {
 
   /** e-book Form */
   if (form_name === 'ebook-form') {
-    const { email, firstName, lastName, company } = data.data;
+    const { email } = data.data;
     const { KLAVIYO_API_KEY, KLAVIYO_LIST_ID_EBOOK } = process.env;
-    const name = `${firstName} ${lastName}`;
 
     await axios({
       url: `https://a.klaviyo.com/api/v2/list/${KLAVIYO_LIST_ID_EBOOK}/subscribe`,
```
Comment (patricskigen) [inline static/functions/submission-created.js:452]: I see some other Klaviyo ID placeholders added to the `sample.env`, should `KLAVIYO_LIST_ID_EBOOK` be added as well?

Comment (naeluh) [reply static/functions/submission-created.js:452]: Sure thanks! Yeah I mean these ids come and go but yes lets do it. thanks!

Comment (patricskigen) [reply static/functions/submission-created.js:452]: Either way, I just saw those other ones getting added to the list

**Review by patricskigen — APPROVED** [review]
LGTM

**Pull request merged**